src/cssLanguageTypes.ts

```diff
@@ -61,9 +61,14 @@ export interface LanguageServiceOptions {
 
 export type EntryStatus = 'standard' | 'experimental' | 'nonstandard' | 'obsolete';
 
+export type MarkupDescription = {
```

Comment:
Why not using the type from vscode-languageserver-types: https://github.com/microsoft/vscode-languageserver-node/blob/db16b7f53b318e8ff44aa95ac8faa5e2486b63aa/types/src/main.ts#L1321